schemas/layer-4.cue

```diff
@@ -1,35 +1,104 @@
 // Top level schema //
 
-"evaluation-plans": [...#EvaluationPlan]
+// Framework Evaluation describes the evaluation of a single framework and its controls.
+#FrameworkEvaluation: {
+    // name of the framework being evaluated, ex: NIST 800-53, CCC, OSPS, etc.
+    // TODO: why materialize/duplicate this data here if a consumer can simply reference the layer 2 data?
+    name: string
+    // frameworkID contains the unique identifier of the framework being evaluated
+    // TODO: in the case of versioned frameworks (ex: NIST 800-53), should we expect the version to be part of the ID?
+    frameworkID: string
+    // one or more evaluations of the framework controls
+    evaluations: [#ControlEvaluation, ...#ControlEvaluation]
+    // the FrameworkEvaluation definition is intentionally left open to accommodate flexible evolution
+    ...
+}
 
-// Types
+// URL describes a specific subset of URLs of interest to the framework
+// TODO: this definition should be imported from a more appropriate module/package
+#URL: =~"^https?://[^\\s]+$"
 
+// ControlEvaluation describes the evaluation of the level 2 control referenced by controlID and the assessment of that control's requirements.
 #ControlEvaluation: {
+    // name of the control being evaluated
+    // TODO: why materialize/duplicate this data here if a consumer can simply reference the layer 2 data?
```

Comment:
Specifically, the `name` field here is a property of the layer 2 control, not of the `ControlEvaluation` and therefore I'd like to remove it from this definition and rely on consumers following the reference to the layer2 control data if they need to name.